filetracker/client/remote_data_store.py

```diff
@@ -116,7 +117,8 @@ def add_file(self, name, filename, compress_hint=True):
                     with gzip.GzipFile(fileobj=tmp, mode='wb') as gz:
                         shutil.copyfileobj(f, gz)
                     tmp.seek(0)
-                    headers.update({'Content-Encoding': 'gzip'})
+                    headers['Content-Encoding'] = 'gzip'
+                    headers['Logical-Size'] = str(os.stat(filename).st_size)
```

Comment:
Non-standard headers should begin with X-

Reply:
https://tools.ietf.org/html/rfc6648
We also have `SHA256-Digest` header already.

Reply:
@accek, do you think it's fine?

Reply:
Oh, I didn't know that one. Looks fine then.